cli/azd/pkg/apphost/testdata/TestAspireDockerGeneration-nodeapp.snap

```diff
@@ -15,7 +15,7 @@ properties:
       allowInsecure: false
     registries:
     - server: {{ .Env.AZURE_CONTAINER_REGISTRY_ENDPOINT }}
-      identity: {{ .Env.AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID }}
+      identity: {{ .Env.AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID }}      
```

Comment:
NIT: Possibly some trailing whitespace got added in the template leading to this diff?